source/operations/network-encryption.rst

```diff
@@ -30,8 +30,9 @@ Enabling TLS
    The TLS certificate generation process is as follows:
 
    - The Operator generates a Certificate Signing Request (CSR) associated to the Tenant.
-     The :abbr:`CSR (Certificate Signing Request)` includes the appropriate DNS SubjectAlternateNames (SAN) for the Tenant services and pods.
-     
+     The :abbr:`CSR (Certificate Signing Request)` includes the appropriate DNS Subject Alternate Names (SANs) for the Tenant services and pods.
```

Comment:
I _think_ `DNS Subject Alternate Names (SANs)` is the most common way to reference this concept?

Reply:
It probably is - and hopefully the majority of users would understand intent here.